sdk/servicebus/service-bus/package.json

```diff
@@ -108,9 +108,9 @@
     "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
     "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
     "build:samples": "node .scripts/prepSamples.js && cd samples && tsc -p .",
-    "build:test:browser": "tsc -p . && cross-env BROWSER_TEST=true rollup -c 2>&1",
-    "build:test:node": "tsc -p . && rollup -c rollup.test.config.js 2>&1",
-    "build:test": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
+    "build:test:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c rollup.test.config.js 2>&1",
+    "build:test:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
+    "build:test": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1 2>&1 && cross-env ONLY_BROWSER=true rollup -c rollup.test.config.js 2>&1",
```

Comment:
Wouldnt `"build:test": "tsc -p . && rollup -c rollup.test.config.js 2>&1` do the same?

Reply:
Agree, updated